src/main/resources/templates/index.html

```diff
@@ -3,59 +3,18 @@
 <head>
     <meta charset="UTF-8">
     <title>Market Data</title>
-    <style>
-        body{font-family: Consolas,serif;}
-        .panel{ width: 400px; height: 400px; border: solid black 1px}
-    </style>
```

Comment:
I moved the CSS to an external file. As your site grows, your CSS will too. You don't want to have a ton of CSS clogging up index.html. Better to move it now rather than later.

Reply:
Definitely... good idea!   HTML should only be content.  Keep styling separate.  Both files will be smaller then.  Less code == less maintenance.